src/main/java/use_case/CreateComment/CreateCommentInputBoundary.java

```diff
@@ -0,0 +1,13 @@
+package use_case.CreateComment;
+
+
+public interface CreateCommentInputBoundary {
+
+    /**
+     * Executes the create comment case.
+     * @param CreateCommentInputData the input data
+     */
+    void execute(CreateCommentInputData CreateCommentInputData);
```

Comment:
lets name this method `createComment` (rename things implementing/using this as well)